ffms2-sys/README.md

```diff
@@ -0,0 +1,5 @@
+# ffms2-sys
+Automatically generated bindings for FFMS2.
+
+## Building
+This crate requires an installed version of FFMS2 on the build system. If possible, it will use *pkg-config* to locate and link the libraries. Otherwise, it could utilize the environment variables FFMS_INCLUDE_DIR and FFMS_LIB_DIR to locate the required files. This strategy is especially helpful on Microsoft Windows (where you, however, likely have to set LIBCLANG_PATH, too).
```

Comment:
Could you please provide a sequence of steps that explain to a user which environment variables need to be set up and how you compile `ffms2.dll` on `Windows`? 

Perhaps we can split up the `README` in two headers, one that explains how to compile on Linux and the other one how to compile on Windows.

Reply:
What do you think of: 

This crate requires an installed version of FFMS2 on the build system. If possible, it will use *pkg-config* to locate and link the libraries. Otherwise, it could utilize the environment variables FFMS_INCLUDE_DIR and FFMS_LIB_DIR to locate the required files specified by the user.

This strategy is especially helpful on Microsoft Windows. In this case, one does not even have to compile the library at all. Instead, developers could refer to the [official releases](https://github.com/FFMS/ffms2/releases). FFMS_INCLUDE_DIR corresponds to the "include" directory while FFMS_LIB_DIR must point to "x64" or "x32", respectively. For the final distribution of the final artifact, the corrresponding *.dll must be shipped beside the executable.

Reply:
I like this description! Please add it into a new markdown heading called `Windows build`, thanks!